frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -226,64 +300,88 @@ <h3 class="input-title location-title">City</h3>
-                  (input)="onStep1Input('state', $event)"></textarea>
+                type="text"
+                class="location-input"
+                rows="1"
+                [value]="state.step_1.state"
+                (input)="onStep1Input('state', $event)"
+              ></textarea>
             </div>
             <div class="location-title-container">
               <h3 class="input-title location-title">Country</h3>
             </div>
             <div class="location-card app-card">
               <textarea
-                  type="text" class="location-input" rows="1" [value]="state.step_1.country"
-                  (input)="onStep1Input('country', $event)"></textarea>
+                type="text"
+                class="location-input"
+                rows="1"
+                [value]="state.step_1.country"
+                (input)="onStep1Input('country', $event)"
+              ></textarea>
             </div>
             <div class="location-title-container">
               <h3 class="input-title location-title">Venue</h3>
             </div>
             <div class="location-card app-card">
               <textarea
-                  type="text"
-                  class="location-input"
-                  rows="1"
-                  [value]="state.step_1.venue"
-                  (input)="onStep1Input('venue', $event)"></textarea>
+                type="text"
+                class="location-input"
+                rows="1"
+                [value]="state.step_1.venue"
+                (input)="onStep1Input('venue', $event)"
+              ></textarea>
             </div>
           </div>
           <div class="description-container">
             <h3 class="input-title">Description</h3>
             <div class="description-card app-card">
               <textarea
-                  type="text"
-                  class="description-input"
-                  rows="2"
-                  [value]="state.step_1.description"
-                  (input)="onStep1Input('description', $event)"></textarea>
+                type="text"
+                class="description-input"
+                rows="2"
+                [value]="state.step_1.description"
+                (input)="onStep1Input('description', $event)"
+              ></textarea>
             </div>
             <div class="segment-list app-card">
               <div
-                  *ngFor="let segment of state.step_3.segments; let i = index"
-                  class="segment-container"
-                  [style.margin-left.rem]="step2Data[i].siblingId > 0 ? 4 : 1">
-                <h3 class="title">{{segment['name']}}</h3>
+                *ngFor="let segment of state.step_3.segments; let i = index"
+                class="segment-container"
+                [style.margin-left.rem]="step2Data[i].siblingId > 0 ? 4 : 1"
+              >
+                <h3 class="title">{{ segment["name"] }}</h3>
```

Comment:
We have tried to standardize around the format {{xx}} instead of {{ xx }}. Personally, I am OK with {{ xx }} but Mikhail seems to really like {{xx}}.